src/PostgREST/Query/SqlFragment.hs

```diff
@@ -213,6 +214,9 @@ pgFmtField table (c, []) = SQL.sql (pgFmtColumn table c)
 -- "operator does not exist: json = unknown"
 pgFmtField table (c, jp) = SQL.sql ("to_jsonb(" <> pgFmtColumn table c <> ")") <> pgFmtJsonPath jp
 
+pgFmtBodyFilter :: QualifiedIdentifier -> QualifiedIdentifier -> FieldName -> SQL.Snippet
+pgFmtBodyFilter table cte f = SQL.sql (pgFmtColumn table f <> " = " <> pgFmtColumn cte f)
```

Comment:
Since this function is only used on `mutatePlanToQuery`, how about scoping it to just that function?

Reply:
Ah, I was planning to reuse it for bulk DELETEs. But I could leave it inside UPDATE for now and, if needed, modify it in the other PR.